src/quo/components/wallet/summary_info/view.cljs

```diff
@@ -30,13 +30,16 @@
 
 (defn networks
   [values theme]
-  (let [{:keys [ethereum optimism arbitrum]} values
-        show-optimism?                       (and optimism
-                                                  (or (pos? (:amount optimism))
-                                                      (= (:amount optimism) "<0.01")))
-        show-arbitrum?                       (and arbitrum
-                                                  (or (pos? (:amount arbitrum))
-                                                      (= (:amount arbitrum) "<0.01")))]
+  (let [{:keys [ethereum optimism arbitrum base]} values
+        show-optimism?                            (and optimism
+                                                       (or (pos? (:amount optimism))
+                                                           (= (:amount optimism) "<0.01")))
+        show-arbitrum?                            (and arbitrum
+                                                       (or (pos? (:amount arbitrum))
+                                                           (= (:amount arbitrum) "<0.01")))
+        show-base?                                (and base
+                                                       (or (pos? (:amount base))
+                                                           (= (:amount base) "<0.01")))]
```

Comment:
It's strange we're adding changes to quo components when adding a new chain, IMO it should be more generic, but it's a larger issue with quo.

For now though, ideally we would want to minimise "chain-specific" code to make it easier to add other ones in the future. This check could be extracted into a function or a separate component, since the operation is the same for each chain.